mautic/mautic
# Fix: Focus cookie path

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Just add path to cookie prevent display item more time as neccessary

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create Focus exit  popup with one day engage timeout
2.  Add Focus to your website
3: Go to  yourtrackingwebsite.tld/page/page2/my-page
4: Then go to yourtrackingwebsite.tld
5. See both activate Focus
6. See also cookies with two mautic_focus_id cookie  - one with /page/page2/my-page path and second with / path

#### Steps to test this PR:
1.  Apply PR and re-save Focus (prevent cache issue)
2.  Repeat steps and see your focus show just one time
